.github/workflows/unit-tests.yaml

```diff
@@ -36,5 +31,10 @@ jobs:
       - name: Install package dependencies.
         run: make install
 
+      - name: Setup service containers
+        run: |
+          docker compose -f docker-compose.test.yml pull
+          docker compose -f docker-compose.test.yml up --detach
```

Comment:
Yeah, I think it would probably make the most sense to have a make command that runs something similar so we can recreate this easily locally with out needing to read this workflow.

Reply:
Already have in ae497a873ceaf58b8d2e28e0b09e66b429b3c364.

Reply:
Oh I forgot to update this after that commit, will update now.